### ISSUE TITLE
base.min.css

### ISSUE THREAD

**Issue by johack0** [Daemonite/material#1]
I have a problem with this file base.min.css. It seems that is not formatted correctly.

**Comment by sesemaya** [comment]
hi @johack0 can you please provide a bit more information about what exactly is the issue/problem?

**Comment by mvaso85** [comment]
something is wrong.. searching..

**Comment by sesemaya** [comment]
closed as a duplicate of https://github.com/Daemonite/material/issues/5

**Comment by johack0** [comment]
the browser does not format the site for nothing

**Comment by sesemaya** [comment]
@johack0 
does the console report an error of getting `base.min.css`? by default, the html page will look for the css file located at `/css/base.min.css`, you may need to change this in the `<head>` accordingly based on your specific setup.

**Comment by mikeworker88** [comment]
https://github.com/mikeworker88/material/blob/patch-1/js/src/wave.js

**Comment by Ugarz** [comment]
Hi,
I have problem a bit similar that I've described [on your forum](http://labs.daemon.com.au/t/dev-wp-navigation-bars-does-not-show-on-click/84).

> ## Context
> Hi,
I discovered the Daemonite Material repo' on Github. I decided to play with it and to do some > templates for websites. I'm trying to adapt Daemonite with in a Wordpress.
> 
> ## Problem
For now, I'm  able to display my own css and Javascript but when I'm trying to open the left nav nothing is poping out. I'm getting an error with the base.min.js with `Uncaught TypeError: $ is > not a function`.
Basically it's [an issue from jQuery](http://stackoverflow.com/questions/12343714/typeerror-is-not-> a-function-when-calling-jquery-function) when we try to use `$` instead of `jQuery`.
> 
> ## Demand
> - Does someone already tried to adapt this repo with Wordpress and get through this ?
> - Have you any tips / tricks to get rid of this issue and display the right and left navbar ?

**Comment by modius** [comment]
From the forum...

This is probably has been solved in the latest Material as the `base.js` no longer includes a copy of jQuery (jQuery is still needed but is loaded separately).

If you are still having the same issue after update Material to the latest version, you may need to check this page out: 

https://learn.jquery.com/using-jquery-core/avoid-conflicts-other-libraries/3
